data/deep missions.txt

```diff
@@ -437,7 +436,18 @@ mission "Deep: Mystery Cubes 2"
 			variant
 				"Cruiser"
 				"Gunboat" 2
-		
+	npc
```

Comment:
The reason I didn't have the fighters in this missions is because the fleet realistically isn't going to use their fighters for something other than what their suppose to, and in this case that is blockading Nihal. Should the fighters die in some incident while traveling from Sol to Nihal, it makes no sense for them to suddenly reappear.

There are already example in-game of fighters not being part of the fleet in one missions but being part of the fleet in the next (e.g. when you bring an Aerie, Bactrian, and Cruiser into Syndicate space to kidnap that executive. On the way there there are no fighters or drone but on your way out there are).